torch/jit/__init__.py

```diff
@@ -369,14 +369,23 @@ def _script_graph(fn, _frames_up=0):
 
 def script(fn, _frames_up=0):
     graph = _script_graph(fn, _frames_up=_frames_up + 1)
-    return torch._C.GraphExecutor(graph, True)
+
+    fn_attributes = {}
+    for i in functools.WRAPPER_ASSIGNMENTS:
+        fn_attributes[i] = getattr(fn, i)
```

Comment:
Can you please rewrite this as a dict comprehension? Also, can't we just use `functools.wraps`?

Reply:
`functools.wraps` does not work with python2 because `__doc__` of `type` is not writable.